tests/css_check_test.js

```diff
@@ -26,38 +48,33 @@ const getAttribute = async (string) => {
            return res.body;
         }else throw new Error('Something is wrong: '+res);
             
-    })
-    const attr = response.split(' ')[1].split('{')[1].split(":")[0]; 
-    const value = response.split(' ')[1].split('{')[1].split(":")[1].slice(0, -1); 
-    const res = { attr:attr, value:value };
-    return res;
-}
-
-const basicRBG = {
-    red : 'rgb(255, 0, 0)',
-    yellow : 'rgb(255, 255, 0)',
-    grey : 'rgb(128, 128, 128)',
-    blue : 'rgb(0, 0, 255)',
-    green : 'rgb(0, 128, 0)'
+    });
+    if(response){
```

Comment:
I'm more concerned with verifying the active keys, that the changes are applied, and that correct events are sent for confirmation - less so about checking the allocation request itself. We have unit tests to cover many details of the specific repos. The goal of this project is to ensure the repos work correctly together in achieving the above things